### ISSUE TITLE
[bug-website module] List of all English problem sites

### ISSUE THREAD

**Issue by molitar** [dazedcat19/FMD2#124]
**Prerequisite**
- [x ] I'm using the latest version.
- [ ] All website modules are up to date.

- [x ] This is not temporary server issue.

**System information**
Windows 10

I spent two complete days testing out all sites under the category of: Adult; English; English-Scanlation; H-Sites; Web Comics 

I hope we can get rid of the trash sites and definitely gone ones now and clean up FMD for English sites. 

**TRASH DOMAINS** (missing chapters and/or incorrect chapters)
--------------
Holymanga
Heavenmanga
Koomanga
MangaZuki
MangaZukiMe (lot of chapters redirect to other sites)
MangaZukiOnline (redirects to shut down site kissmanga)
MangaZukiSite (many redirects)
MangaTurf (lack of resources so unable to access site)
PowerManga (No updates on it in 2 years and only 2 incomplete mangas)

**UNABLE TO LIST CHAPTERS** (chapters show 0)
------------------------
MangaFreak
MangaReader
ManhwaTime
SenseScans
Shoujohearts
Shoujosense
SleepyPandaScans

**UNABLE TO DOWNLOAD LIST** (Site is live)
-------------
MangaGoMe
Rocaca
TeabeerComics
AntisenseScans
AsuraScans
HeroManhua
ManhwaTime
AHStudios
ManhuasWorld
ReadKomik
SekteKomik
Tapas
HentaiHand
HentaiRead
Luscious
MyReadingManga

**EXPECTED DOWN**
-------------
ecchiscans.com
mangafull.org (recently registered only 9 days old seems to have been sold and bought)
MangaRoom (At a park page possibly pulled)
MangaWeek (redirects to another domain if you use http only)
BoredomSociety (At a park page possibly pulled)
BunnysScans
CanisMajorScans
BaekjinScans (URL changed and that new URL is now for sale)
OneShotScans (site gone shows only copyright page)
NManhwa (down for a month)

**CURRENTLY DOWN** (Status Unknown if temp or not)
--------------
ReadComicBooksOnline
SenManga (Bad Gateway)
YoManga
FujoshiBitches
hatigarmscanz (Under maintenance so site format will most likely be changed)
Manhuas
ManhwaPool
TraduccionesAmistosas (last update by cutestat is over 2 months)

**DEFINITELY GONE**
----------------
Kissmanga (shut down for DMC)
ReadMangaEU (domain for sale)
MangaReaderOnline (domain for sale)
ReadRawManga (Account suspended)
EGScans (now redirects to mangadex)
ForgottenScans  (No update since June on on blog reader site down)
HoshikuzuuScans (No update since June on on blog reader site down)
Jaiminisbox (reader gone all updates now on mangaplus)
MangaichiScan  (No update since 10/19 on on blog reader site down)
MangaStream  (No update 9 months on on reddit site down)
PhoenixSerenade (No update in 2 years reader site down)
PsychoPlay (Updates are only posted on MangaDex now)
Riceballicious (3 year old site no longer online)
RoseliaScanlations (absolete no longer in use)
S2Scans (Reader offline site not updated in 3 years)
SeaOtterScans (Reader offline site not updated in 3 years)
TrashScanlations (absolete no longer in use 2 years since update)
TwistedHelScans (Updates are only posted on MangaDex now)
WhiteCloudPavilion (Updates are only posted on MangaDex now)
WorldThree (site inactive since Jan and reader offline)
YaoiIsLife (absolete no longer in use 2 years since update - really did not want to test this one)
GetManhwa (reported gone by site informer)
PhenomenalNoFansub (redirects to a sports site now)
PocketAngelScan (No update in 2 years reader site down)
WoweScans (No update in 2 years reader site down)
DoujinYosh (closed announcement)
ManhwaHand (redirects to porn site)
ReadHentaiManga (down for 4 months according to site informer)
ShosetsuManga (down for 8 months according to site informer)

**NEW URL**
----------------
EarlyManga (earlymanga.xyz)

**Comment by dazedcat19** [comment]
Please make a pull request. We are short of people.

**Comment by Slasar41** [comment]
@dazedcat19 Should we remove all that definitely gone?

**Comment by molitar** [comment]
I would ask to remove the trash domains also. I have done extensive testing and not a single manga has all chapters  Start to read a chapter and missing images.  This also causes failed downloads in FMD.  It's really not worth keeping these as it just causes reports that have nothing to do with FMD.  There is a lot of good reliable sites with same manga so why should we keep ones that are of poor quality?

**Comment by dazedcat19** [comment]
@Slasar41 yes. we can always pull it out of git history later if necessary.

**Comment by molitar** [comment]
Since don't want to remove trash sites.. Would it be possible to have some sort of exclusion list so they never show in our list?  Since sites only down and may come back I need to check them later to see if back online.  But the trash sites I never want to ever even test them trash sites again.  Is there maybe some sort of method that in the website selection we can permanently hide or maybe a check box for our own ignore?

**Comment by dazedcat19** [comment]
Just ignore it then. The website selection list exist for this reason. You just select what you need to appear in mangalist selection.

**Comment by molitar** [comment]
> 
> 
> Just ignore it then. The website selection list exist for this reason. You just select what you need to appear in site list selection.

The problem is unless I manually look at this list EVERY single time to see if the other sites have come back up I end up checking them again.  They are trash but no way to mark them as sites I NEVER want to use permanently or to notate.  There is quite a few sites that right now are broken that I do like to use so when there is a LUA update I want to test to see if they are working again or not.   But trying to remember what 9 sites I definitely do not want to check is not easy to remember.

If we had a hide flag than they never show it would make it so much easier to manage our site list.

**Comment by dazedcat19** [comment]
I have difficulty of understanding your comments.
The source is there. You can modify it as you wish. It's not that hard of a language to understand.

**Comment by Slasar41** [comment]
- MangaFreak: Works fine after doing cloudflare workaround.
- MangaReader: Need fixes. (I can't fix it)
- ManhwaTime: Need fixes. (Fixed)
- SenseScans: Need fixes. (Fixed)
- Shoujohearts: Doesn't have reader anymore, redirect to MangaDex.
- Shoujosense: Looks like doesn't have reader anymore.
- SleepyPandaScans: Website died?

**Comment by molitar** [comment]
> 
> 
> I have difficulty of understanding your comments.
> The source is there. You can modify it as you wish. It's not that hard of a language to understand.

Ok let me try to explain again. I periodically check to see if sites that were down are back up again or not. I can't remember to ignore the sites I call trash so end up checking them and than go download again and.. bam bad downloads. The 9 sites are just garbage and not worth keeping.  Holy Manga for example every single manga I use to download from them had bad chapters and checked online and they were bad.  The site does not maintain there manga at all. 

I can't remember how many times I have checked them.. oh site is back up.. find a manga and start to read it to realize the chapters are bad or missing.  Than I say oh yeah that site is bad let me uncheck it.. Couple months later I forget and the process just repeats. 

But it's hard to remember 9 sites when checking to see if the sites are back or not.

**Comment by dazedcat19** [comment]
Still can't understand.
Why don't you just not using it if it was bad. Or report it to get it removed.

**Comment by molitar** [comment]
> 
> 
> Still can't understand.
> Why don't you just not using it if it was bad. Or report it to get it removed.

That is what I did by saying they are no good sites.. aka trash sites.  Slasar41 had made a comment that there is people probaby still using them.  That was my comment back to that reply but it was actually said on discord.  My response was pasted here so I can see after reading the thread why you was confused.  

The management of these sites are lacking they are out of date and bad and missing chapters.  So the trash and definitely down domains should be removed completely. They just cause site bad reports and such when the problem is not FMD but just bad sites.

**Comment by TeepoLurking** [comment]
These are also broken now

- AsuraScans (Requires CF cookies)
- ReaperScans

**Comment by molitar** [comment]
**To Remove**
manhwascan - domain for sale
manhwahand - now belongs to pornhub and redirects there
nmanhwa.com - no longer exists offline for nearly a year
